src/lib/port-finder.ts

```diff
@@ -72,6 +73,26 @@ class PortFinder {
 			this.#unavailablePorts.push( port );
 		}
 	}
+
+	public reclaimUnavailablePort( port?: number ): void {
+		if ( port && this.#unavailablePorts.includes( port ) ) {
+			killPort( port )
+				.then( () => {
+					console.log( `Killed processes using port ${ port }` );
+				} )
+				.catch( ( err ) => {
+					console.error( `Failed to kill processes using port ${ port }: ${ err.message }` );
+				} )
+				.finally( () => {
```

Comment:
These steps are added to the `finally` block as there are some instances where we can have the desired outcome even if `killPort` fails. For example:

- User deletes site, but an error occurs when attempting to kill the port.
- The user's laptop gets put to sleep, thus causing the port to be killed by the OS.
- User returns to laptop and creates another site in the app.
- The reclaimed port is used thanks to the fact it was removed from the `#unavailablePorts` list and `#searchPort` was reset to the default.